lib/usd/utils/util.h

```diff
@@ -49,14 +37,6 @@ namespace MayaUsdUtils {
     MAYA_USD_UTILS_PUBLIC
     bool isInternalReference(const SdfPrimSpecHandle&);
 
-    //! Returns true if the target node has any specs to contribute to the composed prim.
-    MAYA_USD_UTILS_PUBLIC
-    bool hasSpecs(const UsdPrim&);
-
-    //! Returns the layer in composition arc where HasSpecs is set to true
-    MAYA_USD_UTILS_PUBLIC
-    std::vector<SdfLayerHandle> layerInCompositionArcsWithSpec(const UsdPrim& prim);
-
```

Comment:
Cleaned up the usd utility functions. We no longer use them.

UsdPrimCompositionQuery is a nice utility but doesn't deal with SubLayer comp arcs. For the sake of this task, I stopped using it altogether.